include/class.hpp

```diff
@@ -141,7 +147,7 @@ extern ClassDef classes[NUM_CLASSES];
          ? classes[(int)GET_CLASS(ch)].nowear_flag && OBJ_FLAGGED(obj, classes[(int)GET_CLASS(ch)].nowear_flag)        \
          : 0)
 
-enum level_action { LEVEL_GAIN, LEVEL_LOSE };
+    enum level_action { LEVEL_GAIN, LEVEL_LOSE };
```

Comment:
Another mistake.  Looks like your formatter is broken.  It won't break the code, just looks bad.

Reply:
Yeah not sure what happened there